torch/csrc/distributed/c10d/ddp.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <torch/csrc/utils/tensor_flatten.h>
+
+#include <c10d/ProcessGroup.hpp>
+
+#include <ATen/ATen.h>
+
+#include <cstddef>
+#include <vector>
+
+namespace c10d {
+inline void distBroadcastCoalesced(
```

Comment:
Is there any particular reason for this to be inline?

Reply:
The next PR splits this file into a `.h` and `.cpp` so it solves this. For now, the function definition was in the header so I'll need `inline` or `static`